plat/kvm/efi.c

```diff
@@ -518,12 +518,45 @@ static inline void uk_efi_exit_bs(void)
 		uk_efi_crash("Failed to to exit Boot Services\n");
 }
 
+static inline void uk_efi_reset_attack_mitigation_enable(void)
+{
+#ifdef CONFIG_KVM_BOOT_EFI_STUB_RST_ATK_MITIGATION
+	/* The UTF-16 encoding of the "MemoryOverwriteRequestControl" string */
+	char var_name[] = "M\0e\0m\0o\0r\0y\0O\0v\0e\0r\0w\0r\0i\0t\0e\0R\0e"
+			  "\0q\0u\0e\0s\0t\0C\0o\0n\0t\0r\0o\0l\0";
+	uk_efi_uintn_t data_sz;
+	uk_efi_status_t status;
+	__u8 enable = 1;
+
+	status = uk_efi_rs->get_variable((__s16 *)var_name,
+					 MEMORY_ONLY_RESET_CONTROL_GUID,
+					 NULL, &data_sz, NULL);
+	/* There is either no such variable in the firmware database, or no
+	 * variable storage is supported
+	 */
+	if (status == UK_EFI_UNSUPPORTED || status == UK_EFI_NOT_FOUND)
```

Comment:
I take it that these are these not `unlikely` because what is described in the comment above is a common case?

Reply:
Yes, it is quite a common case. By default, default distribution installed `OVMF` does not have such variables enabled, but you can configure it to be if you manually build `OVMF`, but I doubt that is the case for your usual user. Many baremetal machines do have it though.